front/app/resources/GetBlockedUserCount.tsx

```diff
@@ -0,0 +1,55 @@
+// Libraries
+import React from 'react';
+import { Subscription } from 'rxjs';
+import { blockedUsersCount } from 'services/users';
+import { isNilOrError } from 'utils/helperUtils';
+
+interface InputProps {}
+
+type children = (
+  renderProps: GetBlockedUserCountChildProps
+) => JSX.Element | null;
+
+interface Props extends InputProps {
+  children?: children;
+}
+
+interface State {
+  count: number | undefined | null | Error;
+}
+
+export type GetBlockedUserCountChildProps = number | undefined | null | Error;
+
+export default class GetBlockedUserCount extends React.PureComponent<
```

Comment:
Might be better to write a hook instead since we are sun-setting this flow. Something like `useUser` if we are not moving to react-query in this one